jenkins-build-pack-lambda/Jenkinsfile

```diff
@@ -360,20 +366,22 @@ def buildLambda(String runtime) {
 		sh "pip install -r requirements.txt -t library"
 		sh "touch library/__init__.py"
 		// create virtual environment and install pytest
-		sh """
+		sh ""
+		"
 		pip install virtualenv
 		virtualenv venv
-		. venv/bin/activate
+			.venv / bin / activate
```

Comment:
revert this change

Reply:
Did the IDE or some extension do this for you? Sini also had a similar change on her PR and am wondering what is triggering this? What IDE and extensions are you using for indentation/formatting?

Reply:
Using extension (https://marketplace.visualstudio.com/items?itemName=HookyQR.beautify) for indentation fix.

Reply:
Reverted the changes